main.go

```diff
@@ -84,13 +84,34 @@ func main() {
 		os.Exit(1)
 	}
 
+	leaseDuration, err := kedautil.ResolveOsEnvDuration("KEDA_LEADER_ELECTION_LEASE_DURATION", time.Duration(15)*time.Second)
```

Comment:
Optionally, I could change this to no longer provide a default and rely on `*time.Duration` such that the controller manager instead picks the default. These defaults were chosen based on their docs

Reply:
Instead, we could also move any defaults to the chart

Reply:
Yeah, I'd actually prefer if we rely on controller manager defaults.